From 95dbaac729e71dd7d6510e21f9c0e72eedb11ad5 Mon Sep 17 00:00:00 2001
From: thopre01 <thopre01@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Mon, 4 Jan 2016 05:40:53 +0000
Subject: [PATCH 1/3] 2016-01-04  Thomas Preud'homme 
 <thomas.preudhomme@arm.com>

    gcc/
    * doc/install.texi (--with-multilib-list): Describe the meaning of the
    option for arm*-*-* targets.

git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/trunk@232048 138bc75d-0d04-0410-961f-82ee72b054a4
---
 gcc/ChangeLog        |  5 +++++
 gcc/doc/install.texi | 12 +++++++++++-
 2 files changed, 16 insertions(+), 1 deletion(-)

diff --git a/gcc/ChangeLog b/gcc/ChangeLog
index 8316552..c7289f2 100644
--- a/gcc/ChangeLog
+++ b/gcc/ChangeLog
@@ -1,3 +1,8 @@
+2016-01-04  Thomas Preud'homme  <thomas.preudhomme@arm.com>
+
+	* doc/install.texi (--with-multilib-list): Describe the meaning of the
+	option for arm*-*-* targets.
+
 2015-12-04  Release Manager
 
 	* GCC 5.3.0 released.
diff --git a/gcc/doc/install.texi b/gcc/doc/install.texi
index d8d3939..e50489b 100644
--- a/gcc/doc/install.texi
+++ b/gcc/doc/install.texi
@@ -1099,9 +1099,19 @@ sysv, aix.
 @item --with-multilib-list=@var{list}
 @itemx --without-multilib-list
 Specify what multilibs to build.
-Currently only implemented for sh*-*-* and x86-64-*-linux*.
+Currently only implemented for arm*-*-*, sh*-*-* and x86-64-*-linux*.
 
 @table @code
+@item arm*-*-*
+@var{list} is either @code{default} or @code{aprofile}.  Specifying
+@code{default} is equivalent to omitting this option while specifying
+@code{aprofile} builds multilibs for each combination of ISA (@code{-marm} or
+@code{-mthumb}), architecture (@code{-march=armv7-a}, @code{-march=armv7ve},
+or @code{-march=armv8-a}), FPU available (none, @code{-mfpu=vfpv3-d16},
+@code{-mfpu=neon}, @code{-mfpu=vfpv4-d16}, @code{-mfpu=neon-vfpv4} or
+@code{-mfpu=neon-fp-armv8} depending on architecture) and floating-point ABI
+(@code{-mfloat-abi=softfp} or @code{-mfloat-abi=hard}).
+
 @item sh*-*-*
 @var{list} is a comma separated list of CPU names.  These must be of the
 form @code{sh*} or @code{m*} (in which case they match the compiler option
-- 
1.9.1

